VERSIONING.md

```diff
@@ -1,93 +1,110 @@
 # oneTBB Versioning Policies
 
-There are several types of "versioning" used by the oneTBB project.
+oneTBB follows a multi-level versioning approach. This document outlines the different types of versioning 
+used in the oneTBB project and how they relate to compatibility and usage.
 
-1. The *oneTBB specification version*
-2. The *official oneTBB library version*
-3. The *library interface version*
-4. *API versioning* used to maintain backwards compatibility
+## Version Types
 
-The oneTBB project uses semantic versioning (https://semver.org/) and as a oneAPI Library,
-the oneTBB project makes commitments around compatibility as described
-[here](https://www.intel.com/content/www/us/en/docs/oneapi/programming-guide/2024-1/oneapi-library-compatibility.html).
-With semantic versioning, a MAJOR version increment generally means an incompatible API change,
-while a MINOR version increment means added functionality that is implemented in a backwards compatible manner.
+oneTBB uses the following versioning types:
 
-## The oneTBB Specification Version
+| Version Type    | Purpose                          | Relevant Macros / Functions                               |
+|-----------------|----------------------------------|------------------------------------------------------------|
+| Specification   | Tracks [specification](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onetbb/source/nested-index) conformance          | `ONETBB_SPEC_VERSION`                                      |
+| Library         | Identifies the release version   | `TBB_VERSION_MAJOR`, `TBB_VERSION_MINOR`, `TBB_VERSION_STRING`, `TBB_runtime_version()` |
+| Interface       | Ensures binary compatibility     | `TBB_INTERFACE_VERSION_MAJOR`, `TBB_INTERFACE_VERSION_MINOR`, `TBB_INTERFACE_VERSION`, `TBB_runtime_interface_version()` |
+| API (Namespace) | Maintains backwards compatibility | Namespaces: `dN` (headers), `rN` (runtime symbols)         |
```

Comment:
The phrases "Ensures binary compatibility" and "Maintains backwards compatibility" are very similar. Perhaps, Interface is better described as "Used to determine binary compatibility" and "API (Namespace)" is "Mechanism used to maintain backwards compatibility".